test/parallel/test-dgram-bind.js

```diff
@@ -31,7 +32,7 @@ socket.on('listening', common.mustCall(() => {
     socket.bind();
   }, common.expectsError({
     code: 'ERR_SOCKET_ALREADY_BOUND',
-    type: Error,
+    type: e.Error,
```

Comment:
I'd rather this not check against the internal errors objects. These need to be recognizable as normal `Error`, `TypeError` and `RangeError` instances.